client/src/lib/components/zzz/ActionButton.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+    import { cn } from '$lib/utils/cn';
+
+    const { class: externalClass = '', action, disabled } = $props();
```

Comment:
Taking a page from React, let's prefer the more industry-standard `className` convention. Also, let's avoid defaulting to an empty string and instead mark the prop as optional.
```suggestion
    const { class: className, action, disabled } = $props();
```